versioning/BRM/extentmap.h

```diff
@@ -1048,11 +1048,10 @@ class ExtentMap : public Undoable
   static const constexpr size_t EM_FREELIST_INITIAL_SIZE = 50 * sizeof(InlineLBIDRange);
   static const constexpr size_t EM_FREELIST_INCREMENT = 50 * sizeof(InlineLBIDRange);
   // RBTree constants.
-  static const size_t EM_RB_TREE_NODE_SIZE = 2 * (sizeof(EMEntry) + 8 * sizeof(uint64_t));
+  static const size_t EM_RB_TREE_NODE_SIZE = sizeof(EMEntry) + 8 * sizeof(uint64_t);
```

Comment:
The node size was taken from the node size on my machine, I think to keep it safe we could add some padding, not sure it's needed currently, but how knows.